internal/super/setup.go

```diff
@@ -91,15 +91,19 @@ func create(
 	pickedScaffold := scaffolds[0]
 
 	if setupFlags.Scaffold {
-		scaffoldList := make(map[string][]string)
-		for _, s := range scaffolds {
-			scaffoldList[s.Type] = append(
-				scaffoldList[s.Type],
-				fmt.Sprintf("%s - %s", output.Bold(s.Name), s.Description),
+		items := make([]util.ScaffoldItem, 0)
```

Comment:
can we rename it back to `scaffoldList` or `scaffoldItems` to be more specific? `items` sounds very generic